forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -4683,8 +4683,18 @@ public final void addChangedCardKeywords(final List<String> keywords, final List
         if (keywords != null) {
             long idx = 1;
             for (String kw : keywords) {
-                kws.add(getKeywordForStaticAbility(kw, staticId, idx));
-                idx++;
+                // CR 113.11
+                boolean canHave = true;
+                for (Keyword cantKW : getCantHaveKeyword()) {
+                    if (kw.startsWith(cantKW.toString())) {
+                        canHave = false;
+                        break;
+                    }
+                }
+                if (canHave) {
+                    kws.add(getKeywordForStaticAbility(kw, staticId, idx));
+                    idx++;
```

Comment:
i think the idx should still be go up

for example if something would give 3 Keywords,
and the first one is "can't have", then the following keywords would get different ids than if the first one is not negated.

that could cause more objects that not used

Reply:
good point